editor/editor_export.cpp

```diff
@@ -699,9 +700,11 @@ Error EditorExportPlatform::export_project_files(const Ref<EditorExportPreset> &
 	Set<String> paths;
 	Vector<String> path_remaps;
 
+	//RESOURCES TAB
 	if (p_preset->get_export_filter() == EditorExportPreset::EXPORT_ALL_RESOURCES) {
-		//find stuff
-		_export_find_resources(EditorFileSystem::get_singleton()->get_filesystem(), paths);
+		//puts into 'paths' the path of every resource in res:// (or whatever it is the get_filesystem() returns)
```

Comment:
Likewise "(or whatever it is the get_filesystem() returns)" is a bit too informal